plugins/depthCamera/src/DepthCamera.cc

```diff
@@ -123,17 +133,16 @@ void GazeboYarpDepthCamera::Load(sensors::SensorPtr _sensor, sdf::ElementPtr _sd
     {
         yCError(GAZEBODEPTH) << "GazeboYarpDepthCamera : error in connecting wrapper and device ";
     }
+    #endif
     
     //Register the device with the given name
     std::string scopedDeviceName;
     if(!m_driverParameters.check("yarpDeviceName"))
     {
-        scopedDeviceName = m_sensorName + "::" + driver_list[0]->key;
-    }
-    else
-    {
-        scopedDeviceName = m_sensorName + "::" + m_driverParameters.find("yarpDeviceName").asString();
+        yCError(GAZEBODEPTH) << "missing yarpDeviceName parameter";
+        return;
     }
```

Comment:
This change does not seem back-compatible, and it affects the code even if `GAZEBO_YARP_PLUGINS_DISABLE_IMPLICIT_NETWORK_WRAPPERS` is OFF .

Reply:
fixed